assignment10/controllers/employeeController.js

```diff
@@ -0,0 +1,83 @@
+var mongoose = require('mongoose'), 
+  employees = mongoose.model('employees')
+  project = mongoose.model('projects');
+
+exports.allEmployees = function(req, res){
+  employees.find({}, {name : 1,username : 1,_id : 0},(err, result) => {
+    if (err != undefined) {
+      res.send(err);
+    }
+    else {
+    res.send(result);
+    }
+  });
+};
+
+exports.removeEmployee = function(req, res){
+  var username = req.params.username;
+  var id = null;
+
+  var removeFromProject = employees.findOne({'username' : username}).exec(function(err,result){
+    if (err) return err
```

Comment:
@abansal26 Can we use ternary operator here? If yes, Then please change the code.

Reply:
Added. Commit ID - 0836b72